src/legacy/ui/public/vis/editors/default/components/default_editor_agg_add.tsx

```diff
@@ -80,7 +80,7 @@ function DefaultEditorAggAdd({
     return count >= schema.max;
   };
 
-  return stats.max > stats.count ? (
```

Comment:
This logic was moved to [DefaultEditorAggGroup](https://github.com/elastic/kibana/pull/40866/files#diff-5196c51faa9492fb15b9d72ffd692ee3R201)